src/style/app.module.css

```diff
@@ -50,6 +50,7 @@
   --black: black;
   --rating-star-filled: #ff6d75;
   --rating-star-hover: #ff3d47;
+  --text-grey-light: #959595;
```

Comment:
Don't use a color in the name of the variable. Describe what it does. Look at the `rating-star` examples above.